salt/dex/init.sls

```diff
@@ -112,7 +112,7 @@ dex_instance:
 kubernetes_roles:
   cmd.run:
     - name: |
-        kubectl apply -f /root/roles.yaml || kubectl apply -f /root/roles.yaml
```

Comment:
This looks really confusing :) I understand that we would like remove it, but first probably we would like to resolve potential race condition as @kiall said.